components/dashboard/src/components/tos/terms-of-service.tsx

```diff
@@ -148,13 +158,15 @@ export class TermsOfService extends React.Component<TermsOfServiceProps, TermsOf
                                     variant='text'
                                     color={'secondary'}
                                     onClick={this.onDecline}
+                                    disabled={this.state.submitted}
```

Comment:
to prevent resubmission if we have a long line to the proxy or dashboard, i.e. make it impossible to resubmit.